tests/scripts/thread-cert/config.py

```diff
@@ -106,6 +106,8 @@
 
 PROTOCOL_VERSION = 2
 
+PORT_OFFSET = int(os.getenv('PORT_OFFSET', '0'))
```

Comment:
as discussed offline, prefer solving this issue in the particular test case only. And use a new solution for migrating to cmake.